include/librealsense2/h/rs_internal.h

```diff
@@ -135,6 +135,7 @@ struct rs2_software_device_destruction_callback;
 
 /**
  * Create librealsense context that will try to record all operations over librealsense into a file
+ * \deprecated
```

Comment:
Please explain what do you mean by deprecated? what will happen if someone calls this method? Will he get an exception? It would be great if this would be written besides the "deprecated" comment

Reply:
ok